modules/system/lang/en/lang.php

```diff
@@ -61,9 +63,11 @@
             'users' => 'Users',
             'system' => 'System',
             'social' => 'Social',
+            'backend' => 'Backend',
```

Comment:
Where did this and the `notifications` language keys come from? Why are they not already in the develop branch's english system language file?